### DISCUSSION TITLE
How can I get Death Ray now?(I like it very much,but I haven't seen it again since 0.15v)

### DISCUSSION THREAD

**Discussion by chenpishui12345** [HasangerGames/suroi#304] [Q&A] [answered]
![image](https://github.com/HasangerGames/suroi/assets/165793345/16ff8ffd-63e6-4de1-ac4b-5808efb01891)
![image](https://github.com/HasangerGames/suroi/assets/165793345/b0f710ac-4353-4cda-80a8-1156ad646b38)

**Comment by chenpishui12345** [comment]
the best gun in suroi..

**Reply by adtcqd** [reply]
Let me guess, you came from China, right?
一看名字就知道是中国的兄弟吧 没猜错你应该叫“陈皮水”（手动狗头）

**Reply by chenpishui12345** [reply]
完全正确（（（

**Reply by chenpishui12345** [reply]
你也是？

**Reply by adtcqd** [reply]
是的！！

**Comment by emeraldn30** [comment]
there is no way you can get it

**Reply by chenpishui12345** [reply]
kill a dev?

**Comment by tlk106** [comment]
it's a dev weapon

**Reply by Affrimation** [reply, marked as answer]
Currently it stands as a developer weapon mainly used for trolling or testing <3
